client/app/components/QueryEditor.jsx

```diff
@@ -208,7 +208,7 @@ class QueryEditor extends React.Component {
                 {this.props.dataSources.map(ds => <option label={ds.name} value={ds.id} key={`ds-option-${ds.id}`}>{ds.name}</option>)}
               </select>
               {this.props.canEdit ?
-                <Tooltip placement="top" title={modKey + ' + S'}>
+                <Tooltip placement="top" title={modKey + ' + S'>}>
```

Comment:
Seems it's invalid jsx (or am I wrong?)

Reply:
It's probably here to test that the build actually fails :)

Reply:
Probably :-) But I think we shouldn't keep it :-)

Reply:
Ooops, yeah, so this was hard to get right and I wanted to have a real-life example to test this with. Will remove once I have this working as expected.